Cargo.toml

```diff
@@ -65,13 +64,15 @@ kanata-tcp-protocol = { path = "tcp_protocol" }
 [target.'cfg(target_os = "macos")'.dependencies]
 karabiner-driverkit = "0.1.3"
 core-graphics = "0.23.2"
+open = "5"
```

Comment:
I would prefer to have the dependency be exclusive to windows+gui-feature for now.

Reply:
ok, made them optional